lib/cc/services/github_issues.rb

```diff
@@ -74,7 +74,8 @@ def create_issue(title, issue_body)
       {
         id: body["id"],
         number: body["number"],
-        url: body["html_url"]
+        url: body["html_url"],
+        message: "Issue <a href='#{body["html_url"]}'>##{body["number"]}</a> created.",
```

Comment:
It looks like HTML is included within other message fields from the `#receive_test` payload. Here's [another example](https://github.com/codeclimate/codeclimate-services/blob/master/lib/cc/services/lighthouse.rb#L28). The content is likely what ends up on codeclimate.com when you click "Test integration". As to whether it was meant for other purposes, I'm not sure. A message of "Success" doesn't seem very useful.

Based on [bryan's comment](https://github.com/codeclimate/browser/pull/147#discussion_r64154644), it might make sense to widen this up. Or we could rely on client-side copy.